ui/base-tool/src/main/kotlin/org/cru/godtools/base/tool/ui/settings/ToolOptionsSettingsBottomSheetDialogFragment.kt

```diff
@@ -137,13 +131,13 @@ class SettingsBottomSheetDialogFragment :
         }
         dismissAllowingStateLoss()
     }
+    // endregion ToolOptionsSettingsSheetCallbacks
+}
 
-    override fun toggleTrainingTips() = activityDataModel.showTips.toggleValue()
-
-    override fun swapLanguages() {
-        val languages = activityDataModel.primaryLocales.value
-        activityDataModel.primaryLocales.value = activityDataModel.parallelLocales.value
-        activityDataModel.parallelLocales.value = languages
-    }
-    // endregion TractSettingsSheetCallbacks
+interface ToolOptionsSettingsSheetCallbacks {
```

Comment:
the name `OptionSettings` is redundant, it should either be `ToolOptions` or `ToolSettings`